content/kubermatic/master/installation/install_kubermatic/_index.en.md

```diff
@@ -211,13 +213,13 @@ to see what's causing the issues.
 Before installing the Kubermatic Operator, the Kubermatic CRDs need to be installed. You can install them like so:
 
 ```bash
-kubectl apply -f charts/kubermatic/crd/
+kubectl apply -f config/kubermatic/crd/
```

Comment:
btw @scheeles, the release package (at least for 2.14) doesn't contain the Kubermatic crd. Those can be found in the GitHub repo. So either we mention in the doc that they need to be grabbed for the Kubermatic repo or we include them in the release files.

Reply:
It's an oversight that they are not included in the release file. I'll fix that.

Also, head's up: The path inside the Kubermatic repo is soon (this weekend, https://github.com/kubermatic/kubermatic/pull/5555) also going to be `charts/` and not `config/`. So for the master documentation, this needs to stay charts/..., but for 2.14 it needs to be config/....

Reply:
Release archive has been fixed.

Reply:
Perfect, I will fix that then

Reply:
@xrstf should be all ok now